lib/turbotrig/turbotrig.cpp

```diff
@@ -233,16 +233,16 @@ float turboatan(float x)
     return M_PI/2.0 - turboatan(1.0/x);
   }
 
-  float t = (x - atan_min_x)/(float)(atan_max_x - atan_min_x) * (float)atan_num_entries;
-  int16_t index = (int)t;
-  float dx = t - index;
+  float t = (x - atan_min_x)/static_cast<float>(atan_max_x - atan_min_x) * static_cast<float>(atan_num_entries);
```

Comment:
`atan_max_x` and `atan_min_x` are both already floats, we shouldn't need this cast